### PR TITLE
Apmspii 912 proxy behaviour intermittent

### PR DESCRIPTION
**Pull request by HenryIwanejko**
## Summary
* Fix intermittent rate limit test

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,6 +36,8 @@ pytest-bdd = "^4.0.1"
 cryptography = "^3.2.1"
 python-dateutil = "^2.8.1"
 pytest-rerunfailures = "^9.1.1"
+api-test-utils = {url = "https://github.com/NHSDigital/apim-test-utils/releases/download/v1.1.8-alpha/api_test_utils-1.1.8a0-py3-none-any.whl"}
```
Comment (paruliansaragi) [inline pyproject.toml:39]: Why did we need to re-version the test utils?

File: tests/functional/conftest.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ async def _product_with_full_access():
     await _set_default_rate_limit(product)
     await product.update_scopes([
         "personal-demographics-service:USER-RESTRICTED",
-        "urn:nhsd:apim:app:level3:",
+        "urn:nhsd:apim:app:level3:personal-demographics-service",
```
Comment (paruliansaragi) [inline tests/functional/conftest.py:34]: Does this make a difference? Is it more realistic?

File: pyproject.toml (hunk 1 above)
Comment (HenryIwanejko) [reply pyproject.toml:39]: Had trouble with sim auth. tried changing utils version, but wasn't related. I'll revert now.

File: tests/functional/conftest.py (hunk 2 above)
Comment (HenryIwanejko) [reply tests/functional/conftest.py:34]: Yeah, its exactly what we define for the PDS scopes. So yeah more realistic

**Review by paruliansaragi — APPROVED** [review]

**Pull request merged**